app/views/users/show.html.erb

```diff
@@ -23,6 +27,26 @@
               <%= @user.last_name %>
             </div>
             <%= @user.email %><br>
+            <% if @user.secondary_name %>
+              <%= @user.secondary_name %><br>
+            <% else %>
+              <%= "Not Available" %><br>
```

Comment:
Instead of doing N/A like the above fields, let's not show anything on the dashboard if secondary and tertiary contacts don't exist